bundle/regal/lsp/completion/providers/rulerefs/rulerefs.rego

```diff
@@ -10,15 +10,13 @@ ref_is_internal(ref) if contains(ref, "._")
 
 default determine_ref_prefix(_) := ""
 
-determine_ref_prefix(word) := word if {
-	word != ":="
-}
+determine_ref_prefix(word) := word if word != ":="
 
 position := location.to_position(input.regal.context.location)
 
 line := input.regal.file.lines[position.line]
 
-last_word := regal.last(regex.split(`\s+`, trim_space(line)))
+word := location.ref_at(line, input.regal.context.location.col)
```

Comment:
This is the actual fix. As an added bonus — and quite a nice one too! — we'll now also get completions for refs even when the ref is not the last thing being typed on the line.